README.md

```diff
@@ -28,8 +28,6 @@ Detailed steps for using the generator are found below.
 Step 1: Clone the project from github.
 
 Step 2: Update placeholder references in the codebase.
-- Replace the `<CODEOWNER>` placeholder in resource files (`aem-component-generator/src/main/resources`) with
-the name of your company.
 - Update the demo config file (`data-config.json`) to your company defaults, removing references to `NewCo`/`newco`
```

Comment:
No sense in this being a bullet list anymore, since the list is just one item :)